src/solver.cpp

```diff
@@ -679,6 +679,8 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
     Register(registry, ++id, Primitive::ReLU, prelu::MultiWeightsBackward{}.SolverDbId());
     Register(registry, ++id, Primitive::ReLU, prelu::SingleWeightBackward{}.SolverDbId());
 
+    Register(registry, ++id, Primitive::Mha, mha::MhaCKFlashAttentionV2Forward{}.SolverDbId());
+
     // IMPORTANT: New solvers should be added to the end of the function!
```

Comment:
Several IDs were added here recently, but the git doesn't show the conflict. Probably the empty line before the comment should be removed. It might also be necessary to add a note that we shouldn't leave an empty line before this comment, as otherwise Git won't detect the conflict.

Reply:
Ill update and take a look.
Thanks for the heads up!

Pretty surprising that it would not detect a conflict since the same lines should have changed regardless of the newline.
I can update the comment if that ends up being the case though.

Reply:
Yea weird, it didn't flag it as a conflict, but did (luckily) resolve it properly in this case. 
Ill update the comment as well.